### PR TITLE
Pin bnb to <=0.41.1

### PR DESCRIPTION
**Pull request by younesbelkada**
Currently the doc builder job fails because of the issue described in https://github.com/TimDettmers/bitsandbytes/pull/859

The fix should be to pin bnb to the previous working version - 0.41.1

cc @lvwerra @mishig25

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "diffusers": ["diffusers>=0.18.0"],
     "deepspeed": ["deepspeed>=0.9.5"],
     "benchmark": ["wandb", "ghapi", "openrlbenchmark==0.2.1a5", "requests", "deepspeed"],
-    "quantization": ["bitsandbytes>=0.41.0"],
+    "quantization": ["bitsandbytes==0.41.0"],
```
Comment (younesbelkada) [inline setup.py:76]:
```suggestion
    "quantization": ["bitsandbytes==0.41.1"],
```

File: setup.py [hunk 2]
```diff
@@ -73,7 +73,7 @@
     "diffusers": ["diffusers>=0.18.0"],
     "deepspeed": ["deepspeed>=0.9.5"],
     "benchmark": ["wandb", "ghapi", "openrlbenchmark==0.2.1a5", "requests", "deepspeed"],
-    "quantization": ["bitsandbytes>=0.41.0"],
+    "quantization": ["bitsandbytes==0.41.1"],
```
Comment (younesbelkada) [inline setup.py:76]:
```suggestion
    "quantization": ["bitsandbytes<=0.41.1"],
```

**Review by lvwerra — APPROVED** [review]

**Comment by younesbelkada** [comment]
No worries! hopefully the fix was straightforward :D

**Comment by skandermoalla** [comment]
@younesbelkada is it fine to assume that bnb can be unpinned as https://github.com/bitsandbytes-foundation/bitsandbytes/pull/859 is merged?

**Pull request merged**